packages/material-ui-lab/src/ToggleButton/ToggleButton.test.js

```diff
@@ -21,6 +22,10 @@ describe('<ToggleButton />', () => {
     classes = getClasses(<ToggleButton value="classes">Hello World</ToggleButton>);
   });
 
+  it('does forward refs', () => {
+    testRef(<ToggleButton value="classes">Hello World</ToggleButton>, mount);
```

Comment:
With @eps1lon work on #14958, we should be able to kill this test class going forward.